modules/homePage/component/v2/FourthJumbotron/FourthJumbotronComponent.tsx

```diff
@@ -0,0 +1,24 @@
+import ButtonPrimary from "../../../../../uikit/buttonV2/ButtonPrimary";
+
+function ForthJumobtronBox({ children, className }: { className?: string; children?: React.ReactNode }) {
+    return <div className={`relative mt-4 h-[236px] max-w-full overflow-hidden rounded-[36px] bg-dark-background-elevated pt-8 pl-8 text-left sm:h-[280px] xl:h-[300px]     ${className || ""}`}>{children}</div>;
```

Comment:
The `border-radius` here should be `24px` (defined in figma), so you can use `rounded-3xl`

Reply:
okay on it